src/_includes/macros/form.html

```diff
@@ -0,0 +1,244 @@
+           >
+  </div>
+{%- endmacro %}
+
+{# ==========================================================================
+
+   form.textarea()
+
+   ==========================================================================
+
+   Description:
+
+   Builds form textarea field
+
+   label: Name of the field.
+
+   helperText:
+
+   required: Whether the field is required. Defaults to false.
+
+   disabled: Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+
+{% macro textarea(label, helperText=false, required=false, disabled=false) -%}
+  {% from 'macros/util/format/url.html' import slugify as slugify %}
+  {% set id_slug = slugify( label ) %}
+  <div class="form-l_col form-l_col-1">
+    <label for="form_{{ id_slug }}">
+      <span class="form-label-header">{{ label }}</span>
+      {%- if helperText -%}
+        <br/><span class="micro-copy micro-copy__large">{{ helperText }}</span>
+      {%- endif -%}
+      {%- if required -%}
+        * <span class="u-visually-hidden">Required</span>
+      {%- endif -%}
+    </label>
+    <textarea class="input__long"
+              id="form_{{ id_slug }}"
+              {%- if required -%}required{%- endif -%}
+              {%- if disabled -%}disabled{%- endif -%}></textarea>
+  </div>
+{%- endmacro %}
+
+{# ==========================================================================
+
+   form.select()
+
+   ==========================================================================
+
+   Description:
+
+   Builds a select for a form.
+
+   label: Name of the field.
+
+   placeholder: Text for the placeholder.
+
+   fields: An array of all the fields to select.
+
+   size: The size of the field. Defaults to full-width.
+          Possible options: 1, 1-2, 1-3
+
+   required: Whether the field is required. Defaults to false.
+
+   disabled: Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+
+{% macro select(label, placeholder, fields, size='1', required=false, disabled=false) -%}
+  {% from 'macros/util/format/url.html' import slugify as slugify %}
+  {% set id_slug = slugify( label ) %}
+
+  <div class="form-l_col form-l_col-{{ size }}">
+      <label class="form-label-header"
+             for="form_{{ id_slug }}">
+          {{ label }}
+          {%- if required -%}
+            * <span class="u-visually-hidden">Required</span>
+          {%- endif -%}
+      </label>
+      <div class="custom-select">
+          <select class="custom-select_select"
+                  data-placeholder="{{ placeholder }}"
+                  id="form_{{ id_slug }}"
+                  {%- if required -%}required{%- endif -%}
+                  {%- if disabled -%}disabled{%- endif -%}>
+              <option value="">{{ placeholder }}</option>
+              {% for field in fields %}
+              <option value="{{ slugify(field) }}">{{ field }}</option>
```

Comment:
Ah, you're using slugify. I thought there'd be an option for value and text, as long as we make the queries match, that'll work.